delatore/sources/awx_api.py

```diff
@@ -87,17 +93,22 @@ async def start(self, stop_event: asyncio.Event):
         await self.client.subscribe(self.params().topic_in)
         await super().start(stop_event)
 
-    async def get_templates(self, filters: dict = None) -> Optional[list]:
+    async def get_templates(self, filters: dict = None, depth: int = 1) -> Optional[list]:
```

Comment:
You don't really need to define type if default is `1`, IDE will think it's int anyway